### ISSUE TITLE
Most Popular list is always the same for 2 months

### ISSUE THREAD

**Issue by smaric91** [markop159/KODI-Popcorn-Time#134]
Hi, I have updated the URL as some of you described and popcorn time worked normally. Lately when I open the Most Popular category the same movies are shown to me. Android Popcorn time works normally. Do somebody have any idea or similar issue?

**Comment by S0und** [comment]
Both this plug in and the popcorn time android app shows the same movies. There is no difference, because there is no new movie releases.

**Comment by sorin-pop** [comment]
I have the same issue. And I don't agree with @S0und . At first, I also thought that it's because of no new releases due to the pandemic etc. But after what months of nothing changing, it started to be a bit dubious. So I installed a Popcorntime app on my phone and had a look and it turns out that there were actually new films, new releases! So I think this is a valid issue.

**Comment by smaric91** [comment]
Hi, @sorin-pop  thanks for the reply. That is what I have tried to say. Plugin for kodi is not working, for more than two months movie headings are the same. I have installed the android popcorn app and there are all the new movies.

**Comment by S0und** [comment]
@sorin-pop 

i can only comment about my own experience: https://imgur.com/a/9Yvl19X

I don't see any difference here

**Comment by sorin-pop** [comment]
@S0und perhaps we are using different android apps... mine has the name HdMovies and in the Popular movies section it shows newer releases.

**Comment by sorin-pop** [comment]
Anyway, I think by now it's quite suspicious that that list has remained the same since months now. Coronavirus and low film output here or there, it can't be that bad that absolutely nothing comes out, nothing happens. (and as we saw, a couple of us could see that actually there are new popular releases). Anybody else experienced this? Or it just me and @S0und and @smaric91 ?

**Comment by S0und** [comment]
@sorin-pop 
The popcorntime api is not updated, so there are no new movies. You can call that suspicious if you want to. And since this plugin uses the same API endpoint as the official PT apps, they both return the same result as i just demonstrated. 

https://discuss.popcorntime.app/t/no-new-movies-tv-arm/871

Also this is the official Popcorntime app: https://popcorntime.app/  You are using something else, and if it not open source you should be worried, because unofficial PT clones are known to be used maliciously.

**Comment by sorin-pop** [comment]
@S0und 
OK, thanks for pointing me to the correct app. (and letting me know that somebody reported the same issue for that app)

So if I understand correctly, this is a problem inherent to Popcorntime and its API, right? In other words, Popcorntime has for some reason stopped refreshing its "Most Popular" section (and that's why this API returns the same result since then)? And basically you are saying that the issue is not here, in this Kodi addon, right? If yes, can you suggest how/where to report this issue so that actually something happens about it?

**Comment by S0und** [comment]
@sorin-pop Yes, exactly. The issue is with the API and not the app itself. 

 I've looked https://yts.mx/ 's own api (which is probably used in HdMovies) and that's up to date. Problem is, the returned data structure is different from popcorn time's own, so the plugin needs to be updated for this API.

**Comment by sorin-pop** [comment]
Thanks. Then we look forward to the solution.

**Comment by mflorezm** [comment]
Hi, guys, the problem with the .sh API that is not updating is well know by the people of https://github.com/popcorn-official (seee the link: https://github.com/popcorn-official/popcorn-desktop/issues/1690).

Nobody seem to be able to fix the error on 'https://tv-v2.api-fetch.sh', so, some of them decided to rewrite the app to use a custom API like https://yts.mx/ or http://popcorn-ru.tk/ 

nevertheless, https://getpopcorntime.is/ is up and working, so, my question is if is there any way to replace the API URL like on https://github.com/markop159/KODI-Popcorn-Time/issues/127, with the api from https://getpopcorntime.is? does anybody knows if their API is the same from .sh that was used by markop159 on KODI-Popcorn-Time addon? i guess is easy becuase they are a fork from https://github.com/popcorn-official, instead of using another website API that would need to change a lot on markop159 on KODI-Popcorn-Time addon.

Thanks,

MFM

**Comment by S0und** [comment]
They use a different json structure compared to the currently used api. So you can't just swap it out.

**Comment by ivan1986** [comment]
getpopcorntime.is is closed api for fork, yts.mx is for films only, ru.tk same compatible api
i recomend you add option for server url and now actual usable api is ru.tk

**Comment by dequeckerp** [comment]
@ivan1986 In the markop159  KODI-Popcorn-Time addon I changed 'https://tv-v2.api-fetch.sh' to 'https://popcorn-ru.tk' but fetching the movie list fails?

**Comment by yunasc** [comment]
> Nobody seem to be able to fix the error on 'https://tv-v2.api-fetch.sh', so, some of them decided to rewrite the app to use a custom API like https://yts.mx/ or http://popcorn-ru.tk/

I don't get it, they can't fix it because (allegedly) they can't access those servers, or they no longer have an expertise to fix the code?

**Comment by SavySB** [comment]
I can confirm that http://popcorn-ru.tk/ is the proper way to fix this issue.
